src/app/pages/team-page/team-page.component.html

```diff
@@ -122,6 +127,19 @@ <h3>Members</h3>
               </button></p>
             </mat-card>
           </span> <!-- members span -->
+          <span *ngIf="teamDetail.teamPresenter.isOwner">
+            <h3>Team Contacts</h3>
+            <span *ngIf="(!teamDetail.teamPresenter.teamContactEmails ||  teamDetail.teamPresenter.teamContactEmails.length==0)">
+              <mat-icon>sentiment_neutral</mat-icon>
+              <p>There are no other team contacts other than team owner...</p>
+            </span>
+            <mat-card *ngFor="let accountEmail of teamDetail.teamPresenter.teamContactEmails">
```

Comment:
Are teamContactEmails actually added to teamPresenter? I cannot see that the `TeamPresenter` interface was extended, but not sure, if it is necessary (would improve readability, even if it is not necessary).

Reply:
export interface TeamPresenter extends Team, that is how we can access the teamContactEmails